app/adapters/alma_adapter/alma_item.rb

```diff
@@ -36,6 +44,10 @@ def composite_location_label_display
       end
     end
 
+    def in_reserve?
```

Comment:
Maybe we should call it `on_reserve?` I think this is more conventional and matches the property name.